account_invoice_refund_link/views/account_invoice_view.xml

```diff
@@ -18,15 +18,15 @@
                 <page
                     name="refunds"
                     string="Refunds"
-                    attrs="{'invisible':[('move_type', 'not in', ['in_invoice', 'out_invoice'])]}"
+                    invisible="move_type not in ('in_invoice', 'out_invoice')"
                 >
                     <field name="refund_invoice_ids" nolabel="1">
                         <tree
                             decoration-info="state == 'draft'"
                             decoration-muted="state == 'cancel'"
                         >
-                            <field name="partner_id" invisible="1" />
-                            <field name="move_type" invisible="1" />
+                            <field name="partner_id" column_invisible="True" />
+                            <field name="move_type" column_invisible="True" />
```

Comment:
Better to use the usual convention:
```suggestion
                            <field name="partner_id" column_invisible="1" />
                            <field name="move_type" column_invisible="1" />
```

Reply:
That was my first thought, too, but when you search the code for column_invisible="1" you get 62 results, and when you search for column_invisible="True" you get 1151 results. 
Of cause for the invisible attribute it is the opposite result ;)

Reply:
OK, weird that they decided to go the opposite direction... I think the copy/paste from a non conventional form has done a lot of damage. Anyway, I still think we should stick to `1` in OCA to save chars in the diffs.

Reply:
Ok, i will change to `1`